assign2/src/model/objects/Vote.ts

```diff
@@ -0,0 +1,24 @@
+import Post from "./Post";
+
+export type VoteDirection = "up" | "down";
+
+export class Vote {
+    postId: number;
```

Comment:
For immutable objects (all your classes should create immutable instances in the end), TypeScript can also help with the readonly keyword (shows an error if you try to mutate a property): https://blog.strongbrew.io/Redux-best-practices/#make-everything-readonly.